src/phoebus_boundaries/phoebus_boundaries.cpp

```diff
@@ -132,29 +138,64 @@ void OutflowInnerX1(std::shared_ptr<MeshBlockData<Real>> &rc, bool coarse) {
         "OutflowInnerX1Prim", nb, domain, coarse,
         KOKKOS_LAMBDA(const int &l, const int &k, const int &j, const int &i) {
           q(l, k, j, i) = q(l, k, j, ref);
-
-          // Enforce u^1 <= 0
-          Real vcon[3] = {q(pv_lo, k, j, i), q(pv_lo + 1, k, j, i),
-                          q(pv_lo + 2, k, j, i)};
-          Real gammacov[3][3];
-          geom.Metric(CellLocation::Cent, k, j, i, gammacov);
-          Real W = phoebus::GetLorentzFactor(vcon, gammacov);
-          const Real alpha = geom.Lapse(CellLocation::Cent, k, j, i);
-          Real beta[3];
-          geom.ContravariantShift(CellLocation::Cent, k, j, i, beta);
-
-          Real ucon1 = vcon[0] - W * beta[0] / alpha;
-
-          if (ucon1 > 0) {
-            SPACELOOP(ii) { vcon[ii] /= W; }
-            vcon[0] = beta[0] / alpha;
-            Real vsq = 0.;
-            SPACELOOP2(ii, jj) { vsq += gammacov[ii][jj] * vcon[ii] * vcon[jj]; }
-            W = 1. / sqrt(1. - vsq);
-
-            SPACELOOP(ii) { q(pv_lo + ii, k, j, i) = W * vcon[ii]; }
-          }
         });
+
+    // TODO(BRR) Is this always unnecessary if BC inside BH?
```

Comment:
We can always get this back with the magic of version control. I say we delete it.

Reply:
OK. Yeah I was pleasantly surprised to see that this apparently doesn't matter.